src/main/python/keyboard_comm.py

```diff
@@ -306,6 +307,8 @@ def reload_layout(self, sideload_json=None):
 
         self.custom_keycodes = payload.get("customKeycodes", None)
 
+        self.midi = payload.get("midi", None)
```

Comment:
Put this under "vial" as it's a vial extension (see where `self.vibl` is currently assigned)

Reply:
DONE!